agda-algebras.agda-lib

```diff
@@ -1,3 +1,3 @@
 name: agda-algebras
-depend: standard-library-1.7
-include: src/
+depend: standard-library-1.7 cubical-0.3
```

Comment:
why is the library now depending on cubical? I don't think it should involve cubical at all - that would be a different (but still very cool) library!